fmf/base.py

```diff
@@ -517,7 +517,7 @@ def node(reference):
         Keys supported in the reference:
 
         url .... git repository url (optional)
-        ref .... branch, tag or commit ('master' by default)
+        ref .... branch, tag or commit (default branch by default)
```

Comment:
```suggestion
        ref .... branch, tag or commit (default branch if not present)
```

Reply:
IMO otherwise can be read that 'default branch' is a possible value. Or at least for me:)

Reply:
I see, `provided` sounds good as well?